util/util.go

```diff
@@ -432,3 +433,33 @@ var (
 	isUnified     bool
 	isUnifiedErr  error
 )
+
+//FileExistsAndNotADir - Check to see if a file exists
+//and that it is not a directory.
+func FileExistsAndNotADir(path string) bool {
```

Comment:
Does this need to be public?

Reply:
When I was first putting this together yes as I'd the `FindLocalRuntime()` function in run_linux.go.  After I moved this I thought this function might prove to be useful elsewhere at some future time, but since it's not being used yet I probably ought to make it private.